frontend/packages/dev-console/integration-tests/features/Knative/create-event-sources.feature

```diff
@@ -11,7 +11,12 @@ Feature: Create event sources
    Scenario: Different event source enters display in event sources add page : Kn-07-TC03, Kn-08-TC02
       Given user is at Add page
       When user clicks on "Event Source" card
-      Then user will be redirected to page with header name "Event Sources"
+      Then user will get redirected to Developer Catalog page
+      And user will see header name "Event Sources"
+      And user will see the list of Providers
```

Comment:
@gajanan-more will the same logic applies here that the event sources will be installed from different providers